react/features/base/dialog/components/NonReduxDialog.web.js

```diff
@@ -0,0 +1,253 @@
+import AKButton from '@atlaskit/button';
+import AKButtonGroup from '@atlaskit/button-group';
+import ModalDialog from '@atlaskit/modal-dialog';
+import React, { Component } from 'react';
+
+import { translate } from '../../i18n';
+
+/**
+ * Web dialog that uses atlaskit modal-dialog to display dialogs.
+ */
+class NonReduxDialog extends Component {
+
+    /**
+     * Web dialog component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Whether cancel button is disabled. Enabled by default.
+         */
+        cancelDisabled: React.PropTypes.bool,
+
+        /**
+         * Optional i18n key to change the cancel button title.
+         */
+        cancelTitleKey: React.PropTypes.string,
+
+        /**
+         * This is the body of the dialog, the component children.
+         */
+        children: React.PropTypes.node,
+
+        /**
+         * Disables dismissing the dialog when the blanket is clicked. Enabled
+         * by default.
+         */
+        disableBlanketClickDismiss: React.PropTypes.bool,
+
+        /**
+         * Whether the dialog is modal. This means clicking on the blanket will
+         * leave the dialog open. No cancel button.
+         */
+        isModal: React.PropTypes.bool,
+
+        /**
+         * Is ok button enabled/disabled. Enabled by default.
+         */
+        okDisabled: React.PropTypes.bool,
+
+        /**
+         * Optional i18n key to change the ok button title.
+         */
+        okTitleKey: React.PropTypes.string,
+
+        /**
+         * The handler for onCancel event.
+         */
+        onCancel: React.PropTypes.func,
+
+        /**
+         * The handler for the event when submitting the dialog.
+         */
+        onSubmit: React.PropTypes.func,
+
+        /**
+         * Disables rendering of the submit button.
+         */
+        submitDisabled: React.PropTypes.bool,
+
+        /**
+         * Used to obtain translations in children classes.
+         */
+        t: React.PropTypes.func,
+
+        /**
+         * Key to use for showing a title.
+         */
+        titleKey: React.PropTypes.string,
+
+        /**
+         * The string to use as a title instead of {@code titleKey}. If a truthy
+         * value is specified, it takes precedence over {@code titleKey} i.e.
+         * the latter is unused.
+         */
+        titleString: React.PropTypes.string,
+
+        /**
+         * Width of the dialog, can be:
+         * - 'small' (400px), 'medium' (600px), 'large' (800px),
+         * 'x-large' (968px)
+         * - integer value for pixel width
+         * - string value for percentage
+         */
+        width: React.PropTypes.string
+
+    };
+
```

Comment:
So basically you have duplicated the properties from the Abstract dialog? Can we discuss and find some approach and have them in one place so we do not duplicate them.